### PR TITLE
Add Works and about me section

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
-Using Flex and grid box I placed elements in my portfolio well.
- Background image positioning was important for the background images
- I used the list to name my skills and languages learnt

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- PR is descriptive ✔️
- Gitflow flow was correctly used ✔️ 
- Descriptive README file ✔️

## Required Changes ♻️

image

- [ ] Please put all your images in a single directory (Folder).
- [ ] Please follow the design on figma [Figma Design Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -26,28 +26,216 @@ <h3 class="skill">
...
       <div class="illus">
-        <img src="illustration-header-mobile.png" alt="" />
+        <img src="pics/illustration-header-mobile.png" alt="illustration" />
+      </div>
+    </section>
+    <section>
+      <div class="works">
+        <header>
+          <h1 class="recent">MY RECENT WORKS</h1>
+        </header>
```
Comment (elmar8287) [inline index.html:57]: - [ ] Please, consider that you placed "MY RECENT WORKS" into `h1`, as requested by the previous reviewer, but you should not put it inside of the `header` tag. You used the `header` tag at once already, in line [12](https://github.com/MICHAELMUNAVU83/portfolio-mobile/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R12)

File: index.html [hunk 2]
```diff
@@ -26,28 +26,216 @@ <h3 class="skill">
...
+        <h1 class="abtInfo">About Me</h1>
+        <p class="description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <div class="resume">
+          <h1 class="resumeInfo">GET MY RESUME</h1>
```
Comment (elmar8287) [inline index.html:198]: - [ ] Nice job considering the suggestion from the previous reviewer, to use a header for the title, but please, consider also that the `h1` tag could be used one time in your code (website) - for other headings you can use `hx` tags (aka h1, h2, h3, etc.)

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## Hi @MICHAELMUNAVU83 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:

## Optional suggestions
I am marking these changes optional because they were missed by the previous reviewer, I strongly recommend you consider them into action.

- [ ]  Your skills for the Languages, Frameworks are aligned to the left. Please align them to the center and make sure it matches the design on the template.

Your design

Screenshot (92)


Figma design

Screenshot (91)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**